src/test/fixtures/mocks/request-animation-frame.js

```diff
@@ -19,19 +19,93 @@
  *    });
  *    flushRafCalls();
  *
+ * By default this will also repeatedly call requests chained during the
+ * callback, but only 5 times at most, and will throw after that. If you want to
+ * call the callbacks only once, you can pass { once: true } to flushRafCalls:
+ *
+ *    flushRafCalls({ once: true })
+ *
+ * Normally timestamps are passed to the callbacks. By default this mock doesn't
+ * do that, but you can pass the timestamps to use with the "timestamps"
+ * parameter:
+ *
+ *    flushRafCalls({ timestamps: [10000, 10100] });
+ *
```

Comment:
nit: I thought this was a function comment on top of `mockRaf` wouldn't it be better to keep it that way? (e.g. if we add another exported function, we are going to need to move this around)

Reply:
I split the comment in half, put only a small one at the start of the file and kept the big one nearer to the function.